### PR TITLE
Add region attribute to the managed clusters

### PR DESCRIPTION
**Pull request by raghavendra-talur**

### CODE REVIEW

**Comment by raghavendra-talur** [comment]
This should be merged after #365

File: api/v1alpha1/drpolicy_types.go [hunk 1]
```diff
@@ -20,11 +20,17 @@ import (
...
+type Region string
+
 // Managed cluster information
 type ManagedCluster struct {
 	// Name of this managed cluster as configured in OCM/ACM
 	Name string `json:"name"`
 
+	// Region of a managed cluster determines it DR group.
+	// All managed clusters in a region are considered to be in a sync group.
+	Region Region `json:"region"`
```
Comment (ShyamsundarR) [inline api/v1alpha1/drpolicy_types.go:32]: (question) For Metro case, regions have to be equal and an additional zone would be specified?

**Review by ShyamsundarR — APPROVED** [review]
LGTM! waiting for rebase.

**Comment by raghavendra-talur** [comment]
> LGTM! waiting for rebase.

Done.

File: api/v1alpha1/drpolicy_types.go (hunk 1 above)
Comment (raghavendra-talur) [reply api/v1alpha1/drpolicy_types.go:32]: No, we figured that the zone attribute is not required. All clusters that belong to the same region are part of the sync DR group.

**Pull request merged**